cps/web.py

```diff
@@ -1283,7 +1291,7 @@ def get_download_link(book_id, format):
             response.headers["Content-Type"] = mimetypes.types_map['.' + format]
         except:
```

Comment:
Same as before here, kindly ask you to undicate the most specific exception type in ```except``` block.